code-build.js

```diff
@@ -87,16 +90,17 @@ async function waitForBuildEndTime(
     //We caught an error in trying to make the AWS api call, and are now checking to see if it was just a rate limiting error
     if (errObject.message && errObject.message.search("Rate exceeded") !== -1) {
       //We were rate-limited, so add `backOff` seconds to the wait time
-      let newWait = wait + backOff;
+      let newWait = updateInterval + updateBackOff;
```

Comment:
can you add a random jitter to this?  

https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/

Reply:
Applied Full Jitter to the added backoff now. If/when the backoff implementation looks good, I also need to update the backoff description in relevant places.